feat: worker for selfregistersqrttaskmiddleware (#11)
symbioticfi/middleware-sdk: src/examples/sqrt-task-network/worker/README.md

# Sqrt Task Network Worker

This is a worker implementation for the Sqrt Task Network example. The worker listens for new tasks, computes square roots, and submits the results back to the blockchain.

## Prerequisites

- Docker
- Python 3.11+

> - Access to an Holesky Ethereum node

Reviewer: ```suggestion
- Access to a Holesky Ethereum node
```

> - Private keys for validator and operator accounts

Reviewer: How to get operator account (link to docs); and what is private key for validator here

> - Some Holesky ETH in the operator and validator account for gas fees

Reviewer: Need some threshold

> - Registration in `OperatorRegistry` and opt-in in `NetworkOptInService` to the network (middleware address: 0x18586B8cb86b59EF3F44BC915Ef92C83B6BAfd75)

Reviewer: Also, need network address; + link to docs

# Sqrt Task Network Worker

This is a worker implementation for the Sqrt Task Network example. The worker listens for new tasks, computes square roots, and submits the results back to the blockchain.

## Prerequisites

- Docker
- Python 3.11+
- Access to a Holesky Ethereum node
- Private key for validator (used to sign task results and operator registration)
- Private key for operator (used to submit transactions to the network and in core)

> - Some Holesky ETH in the operator and validator account for gas fees

Reviewer: ```suggestion
- Have at least 0.05 Holesky ETH in the operator and validator account each for gas fees
```

> - Registration in `OperatorRegistry` and opt-in in `NetworkOptInService` to the network (middleware address: 0x18586B8cb86b59EF3F44BC915Ef92C83B6BAfd75)

Reviewer: ```suggestion
- Registration in `OperatorRegistry` and opt-in in `NetworkOptInService` to the network ([see example](https://docs.symbiotic.fi/handbooks/operators-handbook#actions-in-symbiotic-core))
    - Network: [0x18586B8cb86b59EF3F44BC915Ef92C83B6BAfd75](https://holesky.etherscan.io/address/0x18586B8cb86b59EF3F44BC915Ef92C83B6BAfd75)
```
